### DISCUSSION TITLE
Vendor Product Listings

### DISCUSSION THREAD

**Discussion by dannybpw** [amzn/selling-partner-api-models#3741] [General]
Hello,

What is the recommended way using the SPAPI to retrieve a full list of currently supported products by a Vendor?

Any API references for retrieving this information seems to be seller only, including the GET_MERCHANT_LISTINGS_ALL_DATA report type.

To re-iterate, looking for Vendor specific.

Thank you in advance!

**Comment by alin-plamadeala** [comment]
Hey @dannybpw,

We are interested in this scenario as well. Did you manage to find a solution?

Thanks

**Reply by egesenli** [reply]
@alin-plamadeala @dannybpw If you find a solution, I would really appreciate it if you could let me know.
